worlds/tunic/__init__.py

```diff
@@ -94,19 +97,24 @@ def generate_early(self) -> None:
         # Universal tracker stuff, shouldn't do anything in standard gen
         if hasattr(self.multiworld, "re_gen_passthrough"):
             if "TUNIC" in self.multiworld.re_gen_passthrough:
-                passthrough = self.multiworld.re_gen_passthrough["TUNIC"]
-                self.options.start_with_sword.value = passthrough["start_with_sword"]
-                self.options.keys_behind_bosses.value = passthrough["keys_behind_bosses"]
-                self.options.sword_progression.value = passthrough["sword_progression"]
-                self.options.ability_shuffling.value = passthrough["ability_shuffling"]
-                self.options.logic_rules.value = passthrough["logic_rules"]
-                self.options.lanternless.value = passthrough["lanternless"]
-                self.options.maskless.value = passthrough["maskless"]
-                self.options.hexagon_quest.value = passthrough["hexagon_quest"]
-                self.options.entrance_rando.value = passthrough["entrance_rando"]
-                self.options.shuffle_ladders.value = passthrough["shuffle_ladders"]
+                self.using_ut = True
+                self.passthrough = self.multiworld.re_gen_passthrough["TUNIC"]
+                self.options.start_with_sword.value = self.passthrough["start_with_sword"]
+                self.options.keys_behind_bosses.value = self.passthrough["keys_behind_bosses"]
+                self.options.sword_progression.value = self.passthrough["sword_progression"]
+                self.options.ability_shuffling.value = self.passthrough["ability_shuffling"]
+                self.options.logic_rules.value = self.passthrough["logic_rules"]
```

Comment:
```suggestion
                try:
                    self.options.logic_rules.value = self.passthrough["logic_rules"]
                except KeyError:
                    warning("The TUNIC APWorld that the multiworld was generated with is newer than the one in your"
                                 "Archipelago installation. Consider updating your APWorld to a newer version.")
```
This option has been removed in a later version. I'd prefer it show an error and continue rather than prevent use of UT with a probably annoying to diagnose error.